modules/isc_dhcpd/lease.go

```diff
@@ -0,0 +1,106 @@
+package isc_dhcpd
+
+import (
+	"errors"
+	"net"
+	"os"
+	"time"
+	"io"
+	"bufio"
+	"bytes"
+	"strconv"
+)
+
+type LeaseFile struct {
+	IP    net.IP
+	Ends  time.Time
+	EndString string
+	State string
+}
+
+func ParseDHCPd(r io.Reader) ([]LeaseFile) {
```

Comment:
`LeaseFile` => `lease` or `leaseEntry`

all fields should be private, why public?

`State` => `bindingState` because it is binding state.

---

shouldnt be public and should be 

`parseDHCPdLeases(leases []lease, r io.Reader) ([]lease)`

Reply:
> all fields should be private, why public?

I forgot the detail that `go` convert to public when variable is Upper case .

Reply:
I changed everything according we talked.